### DISCUSSION TITLE
Detect organization from domain during login

### DISCUSSION THREAD

**Discussion by slabiak** [zitadel/zitadel#7239] [Q&A]
I have two organizations, org1 and org2, and one user user@example.com which belongs to both orgs. I want this user to have different roles in both orgs (basically be a different user). Currently I can create authorization url which by setting `urn:zitadel:iam:org:id:249800091320218895 ` as scope which will point user to login page customized for org 249800091320218895 which is nice. However, I'm wondering if it's possible to have custom login url for each organization? Cause when I enter this auth url with org set as scope i'm redirected to login page with such url:
`https://myzitadelname-f0ko2d.zitadel.cloud/ui/login/login?authRequestID=249912728599136357`
The issue is that user might just copy this url https://myzitadelname-f0ko2d.zitadel.cloud/ui/login/login and enter it in the future, and when it happens such user will be presented login page for default organization where his email address will not work anymore because he might be not registered there and will receive info that user doesnt exist.
I'm wondering if it's possible to have for example https://org1.myzitadelname-f0ko2d.zitadel.cloud/ui/login/login page for org1 and https://org2.myzitadelname-f0ko2d.zitadel.cloud/ui/login/login for org2 so that i dont need to provide org in the scope and that org is resolved based on the domain so that it's not confusing for end user.

**Comment by hifabienne** [comment]
It is possible to have your ZITADEL running on different domains, but at the moment they are not connected to an organization.
Usually it doesn't make sense to store the login url, as with only the login url you do not know which application to open afterwards.
The OIDC request is always started by the app itself.

**Reply by slabiak** [reply]
I agree that you don't know which app to open afterwards but the behaviour could be the same as with login page for default organization. When you enter directly such link https://myzitadelname-f0ko2d.zitadel.cloud/ui/login/login, and perform login, then you are redirected to self service page of default org where you can manage your profile. So the behaviour could be similar. If you dont want to implement it then maybe it woud be a good idea to not allow to enter directly such url https://myzitadelname-f0ko2d.zitadel.cloud/ui/login/login but only via redirect from auth url with whole context, because when you have different orgs and you can enter login page of default org directly and it has the same url as login urls for other orgs it's a bit misleading (at least IMO).
